src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/effect/BurnoutReduction.java

```diff
@@ -1,14 +1,9 @@
 package com.github.minecraftschurlimods.arsmagicalegacy.common.effect;
 
-import com.github.minecraftschurlimods.arsmagicalegacy.common.init.AMBlocks;
-import com.github.minecraftschurlimods.arsmagicalegacy.common.init.AMItems;
 import net.minecraft.world.effect.MobEffectCategory;
 import net.minecraft.world.entity.ai.attributes.AttributeModifier;
 import net.minecraft.world.entity.ai.attributes.RangedAttribute;
 
-/**
- * This class is required so that it doesn't cause classloading issues in {@link AMBlocks} and {@link AMItems}
- */
 public class BurnoutReduction extends AMMobEffect {
```

Comment:
why does this class create an attribute in the constructor? attributes are to be accessed through the registry!